### ISSUE TITLE
Kamera can't scan Kazuha

### ISSUE THREAD

**Issue by sqidleston21** [Andrewthe13th/Inventory_Kamera#106]
I think this is because his name wraps and the scanner doesn't know how to read it. If you skip to the next character and select attributes the tool will continue scanning as normal

**Comment by Cupcak3** [comment]
Unfortunately I don't have any of the long name characters so I couldn't test with any of them extensively. I had a friend send me a screenshot of their window on Kazuha and his name doesn't wrap in either of the places you can see on the attributes screen. Can you send a screenshot of your window with Kazuha selected?

**Comment by Andrewthe13th** [comment]
Could you tell me more details about how the issue occured. Have you using the scanner delay toggle in cause the scanner was a little bit too fast? I just tried the scanner right now and it scanned kazuha just fine. 
![image](https://user-images.githubusercontent.com/11446437/146474917-8b6b6adc-57e6-40b9-88a4-2cd005db4e6a.png)

**Comment by sqidleston21** [comment]
So originally, it would get to Kazuha and would not move with the pointer on attributes every time I scanned (tried maybe 4 or 5 times and each time it got stuck on Kazuha. The navigation didn't seem to be able to pick up his name so I eventually just tried selecting the next character in the list and then the scanner went through every other character just fine. I was using 1600x900 ratio since the update mentioned that we could use that now. My screen for Kazuha is attached below:
![Capture](https://user-images.githubusercontent.com/96272569/146565163-091c0036-608a-4e17-b592-6840c47459df.JPG)

I didn't try slowing the scanner down but I could go try that. However I just went to give it another try this morning and got this error upon scanning my first character (Eula):
Object reference not set to an instance of an object.
   at InventoryKamera.CharacterScraper.ScanCharacters() in B:\Projects\VisualStudio\InventoryKamera\InventoryKamera\InventoryKamera\CharacterScraper.cs:line 23
   at InventoryKamera.InventoryKamera.GatherData(Boolean[] formats, Boolean[] checkbox) in B:\Projects\VisualStudio\InventoryKamera\InventoryKamera\InventoryKamera\InventoryKamera.cs:line 94
   at InventoryKamera.Form1.<>c__DisplayClass21_0.<StartButton_Clicked>b__0() in B:\Projects\VisualStudio\InventoryKamera\InventoryKamera\InventoryKamera\Form1.cs:line 207

The scanner was able to read Eula's name and element before being aborted.

I can go try again with a slower speed to see if that fixes the issue.

**Comment by sqidleston21** [comment]
Update: I just went and tested with a slower delay and it got stuck on Kazuha but this time, produced this error: 

Could not determine character's name or element
Object reference not set to an instance of an object.
   at InventoryKamera.CharacterScraper.ScanCharacters() in B:\Projects\VisualStudio\InventoryKamera\InventoryKamera\InventoryKamera\CharacterScraper.cs:line 23
   at InventoryKamera.InventoryKamera.GatherData(Boolean[] formats, Boolean[] checkbox) in B:\Projects\VisualStudio\InventoryKamera\InventoryKamera\InventoryKamera\InventoryKamera.cs:line 94
   at InventoryKamera.Form1.<>c__DisplayClass21_0.<StartButton_Clicked>b__0() in B:\Projects\VisualStudio\InventoryKamera\InventoryKamera\InventoryKamera\Form1.cs:line 207

**Comment by sqidleston21** [comment]
Sorry for the million comments here, I've just been doing some other testing (part of my irl job and I just like finding things. I can stop after this lol). I changed the display mode back to 1280x720 and everything scanned. 


Only slight issues I saw there were all weapons showing as false for being locked. Almost all of my weapons are locked aside from a few 4*s that I have a lot of duplicates of. When imported into GO, there were slight differences in leveling (but I think that could just be an issue with GO since the file generated from Kamera looks fine). For example: GO shows some weapons as being 70/80 when they're just 70.

Other than that it worked fine with a slower speed and smaller display size.

**Comment by Andrewthe13th** [comment]
I've only tested the scanner in 1280x720. Am sure we still have some small unexpected issues that we need to iron out for the new support resolutions. Thanks for testing.

**Comment by Cupcak3** [comment]
Thanks for the screenshot. I did not expect the name to suddenly wrap to a new line within the same aspect ratio in the top left corner. 

Currently we are not checking for the lock on weapons. We will soon. The lock in GO is purely to prevent accidentally deleting them. 

As for the 70/80 level check, I noticed that sometimes in testing as well. The incorrect 80 is mostly a cosmetic bug as long as the exported file shows the correct ascension level.
